CHANGELOG.md

```diff
@@ -14,19 +33,21 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - ES JSON `uri` now populated using default Orchid item path
 - Tests and fixtures for all supported formats except CustomToEs
 - `get_elements` returns nodeset given xpath arguments
+- `spatial` nested fields `spatial.type` and `spatial.title`
 
 ### Changed
-- Keyword arguments for `get_text`, `get_list`, and `get_xpaths`
-  - This is a breaking change which will require adding keywords such as
-    xml and keep_tags
+- Arguments for `get_text`, `get_list`, and `get_xpaths`
 - XPaths for VRA and TEI to Elasticsearch
 - Default behavior for CsvToEs for some fields
 - Documentation updated
 - Changed Install instructions to include RVM and gemset naming conventions
 - API field `coverage_spatial` is now just `spatial`
 
-### Deprecated
-- `coverage_spatial` API fields
```

Comment:
Should we also list this under "Removed"? The migration step kind of hints at it, but doesn't necessarily mean `coverage_spatial` was removed :thinking:

Reply:
I thought about it, and then decided not to since it was already in Changed and Migration. Also we aren't actually using that field at all anywhere?  I dunno, I'm fine with adding it but I don't feel like we need to, either.

Reply:
Yeah, I think you're right. It'll be clear enough :)